tracing.go

```diff
@@ -484,17 +493,21 @@ func (s *Span) toEvent() *Event {
 		s.dynamicSamplingContext = DynamicSamplingContextFromTransaction(s)
 	}
 
+	contexts := map[string]Context{}
+	for k, v := range s.contexts {
+		contexts[k] = v
+	}
+	contexts["trace"] = s.traceContext().Map()
```

Comment:
This might overwrite the `trace` key, is this a concern?

Reply:
Nope, we want to make sure the `trace` key is always being set by the transaction itself, not the span.